packages/paste-core/components/chat-composer/src/PlaceholderWrapper.tsx

```diff
@@ -0,0 +1,29 @@
+import * as React from 'react';
+import type {BoxProps} from '@twilio-paste/box';
+import {Box} from '@twilio-paste/box';
+
+import type {ChatComposerProps} from '.';
+
+export const PlaceholderWrapper: React.FC<{
+  placeholder: ChatComposerProps['placeholder'];
+  element?: BoxProps['element'];
+}> = ({placeholder, element}) => (
+  <Box
+    position="absolute"
+    top="space0"
+    bottom="space0"
+    display="flex"
+    alignItems="center"
+    color="colorTextWeak"
+    pointerEvents="none"
+    userSelect="none"
+    overflow="hidden"
+    textOverflow="ellipsis"
```

Comment:
question (non-blocking): does textOverflow work on this absolutely positioned element? Also does this need a width: 100% or left/right set?

Reply:
Oh! I can add left/right.

And oof yeah textOverflow doesnt work at all - I'll take it out
<img width="1079" alt="Screen Shot 2022-11-18 at 3 32 47 PM" src="https://user-images.githubusercontent.com/8482334/202820298-f228b696-3fcc-406b-a083-1af3b8945f75.png">